ci/ansible/roles/deploy/defaults/main/packages.yml

```diff
@@ -3,39 +3,37 @@ ansible_dnf_packages:
   - python3-libselinux
   - python3-libsemanage
 dnf_packages:
-  - python3
-  - python3-devel
+  - python3.12
```

Comment:
We reference the Python version (`python3.x`) multiple times. To make this easier to update in the future and less error prone we should put it in the default env vars (EX, `python_version: python3.12`).

This would be used anything `python3.x` is referenced in this commit.

Reply:
I plan to convert it to all pull from a single var in the future but I was trying to stop adding to this PR.